PWGHF/HFL/TableProducer/electronSelectionWithTpcEmcal.cxx

```diff
@@ -365,18 +403,86 @@ struct HfElectronSelectionWithTpcEmcal {
           continue;
         }
 
-        isEMcal = true;
-        std::cout << " electron id  in selection" << electronId << std::endl;
-        electronSel(matchTrack.collisionId(), electronId, etaMatchTrack, phiMatchTrack, ptMatchTrack, pMatchTrack, trackRapidity, matchTrack.dcaXY(), matchTrack.dcaZ(), matchTrack.tpcNSigmaEl(), matchTrack.tofNSigmaEl(),
-                    eMatchEmcCluster, etaMatchEmcCluster, phiMatchEmcCluster, m02MatchEmcCluster, m20MatchEmcCluster, cellEmcCluster, timeEmcCluster, deltaEtaMatch, deltaPhiMatch, isEMcal);
+        /////////////////////////////////////////////////////////////            photonic electron          ///////////////////////////////////////////////////////
```

Comment:
I would suggest to make a new function for invariant mass reconstruction and tagging non-nhfe instead of in the same function "fillElectronTrack"

Reply:
![image](https://github.com/user-attachments/assets/8d033663-3470-4aa3-91e0-a7ef435d9c1c)

To implement different functions, we require two separate structs because inclusive electron analysis is needed for both like-sign and unlike-sign pairs. The tables for these inclusive electron pairs are created in fileElectronTrack, and these tables cannot be used within the same struct. For more information i have attached slide. So please tell me this is correct method or not?

Reply:
I agree with @deepathoms, you can make a new function and can call that function inside your "fillElectronTrack" function if needed

Reply:
I have committed the code after implementing the function. Please review the code and provide your suggestions.